commands/help.rb

```diff
@@ -101,16 +101,16 @@ def self.help(command)
       puts <<~EOT
         Look for package(s).
         Usage: crew search [-v|--verbose] [<pattern> ...]
-        If <pattern> is omitted, all packages will be returned.
```

Comment:
I think we should keep this line to make it clear.

Reply:
This line is no longer accurate.